api-client/src/routes/InternalPathnameParser.ts

```diff
@@ -139,9 +139,10 @@ class InternalPathnameParser {
       return null
     }
 
-    // Single pois are identified via their city content path, e.g. '/augsburg/de/events/1234'
-    const cityContentPath = this._length > ENTITY_ID_INDEX ? this._pathname : undefined
-    return { ...params, route: POIS_ROUTE, cityContentPath }
+    // get the urlSlug from pathname '/testumgebung/de/locations/cafe-tür-an-tür'
+    const urlSlug =
+      this._parts[this._length - 2] === POIS_ROUTE ? decodeURI(this._parts[ENTITY_ID_INDEX]!).toLowerCase() : undefined
+    return { ...params, route: POIS_ROUTE, urlSlug }
```

Comment:
Why is this change needed? The check for `POIS_ROUTE` is definitely unnecessary since it is already done in `this.cityContentParams(POIS_ROUTE)`. Also, I think the `decodeURI` should not be necessary since `decodeURIComponent` is executed in the constructor on the whole pathname.

Reply:
because you have to check the position of "locations" (POIS_ROUTE) to indicate if it has a slug or not after

Reply:
> because you have to check the position of "locations" (POIS_ROUTE) to indicate if it has a slug or not after

Isn't that exactly what `this._length > ENTITY_ID_INDEX` did before?

> does the normalizePath also change apply `toLowerCase()` ?

Sorry but don't understand the question. Imo `toLowerCase` should be run in our `normalizePath` helper function, yes (if that was the question).

Reply:
> > because you have to check the position of "locations" (POIS_ROUTE) to indicate if it has a slug or not after
> 
> Isn't that exactly what `this._length > ENTITY_ID_INDEX` did before?
> 
adjusted it
> > does the normalizePath also change apply `toLowerCase()` ?
> 
> Sorry but don't understand the question. Imo `toLowerCase` should be run in our `normalizePath` helper function, yes (if that was the question).
>
added to `lowerCase()` to `normalize`